libs/community/langchain_community/document_loaders/parsers/doc_intelligence.py

```diff
@@ -15,24 +18,40 @@ def __init__(
         api_endpoint: str,
         api_key: str,
         api_version: Optional[str] = None,
-        api_model: str = "prebuilt-layout",
+        model_id: str = "prebuilt-layout",
```

Comment:
this is breaking change, what's value of doing this?

Reply:
Reverted the API change for this PR.